api/config.go

```diff
@@ -12,14 +23,128 @@ type ConfigUnset struct {
 
 // Config is the structure of an app's config.
 type Config struct {
-	Owner    string                 `json:"owner,omitempty"`
-	App      string                 `json:"app,omitempty"`
-	Values   map[string]interface{} `json:"values,omitempty"`
-	Memory   map[string]interface{} `json:"memory,omitempty"`
-	CPU      map[string]interface{} `json:"cpu,omitempty"`
-	Tags     map[string]interface{} `json:"tags,omitempty"`
-	Registry map[string]interface{} `json:"registry,omitempty"`
-	Created  string                 `json:"created,omitempty"`
-	Updated  string                 `json:"updated,omitempty"`
-	UUID     string                 `json:"uuid,omitempty"`
+	Owner        string                 `json:"owner,omitempty"`
+	App          string                 `json:"app,omitempty"`
+	Values       map[string]interface{} `json:"values,omitempty"`
+	Memory       map[string]interface{} `json:"memory,omitempty"`
+	CPU          map[string]interface{} `json:"cpu,omitempty"`
+	Healthchecks map[string]Healthcheck `json:"healthchecks,omitempty"`
```

Comment:
isn't healthcheck\s json going to be 
```
{
readinessProbe: {}
livenessProbe: {}
}
```
  and httpGet, exec, tcpSocket are just handlers in each in probe